### DISCUSSION TITLE
Additional metadata on requesting callback for large audio files

### DISCUSSION THREAD

**Discussion by pranjal-zipteams** [deepgram/community#62] [General help] [answered]
Hi, I'm using the Python SDK and transcribing very large type files (typically a few hundred MBs).

So far I have successfully:
- Retrieved the transcripts of the calls on a separate endpoint using the `callback` URI
- Made use of the `request_id` and `file_name` mapping created on the origin, to identify and remap transcripts with audio files.

What I want to know:
- Is there a way we can send additional custom `metadata/parameters` while requesting a `callback` to a specific URI?
- If so, could you please point me to the reference of what options to use for this?

Thanks!

**Comment by BrianHillis-DG** [comment, marked as answer]
Hey @pranjal-zipteams, it sounds like the [tag feature](https://developers.deepgram.com/documentation/features/tag/) is what you're looking for.  

### Python SDK Usage: 
``` python
#single tag
response = await dg_client.transcription.prerecorded(source,
        { "model": "general", 
          "language": "en", 
          "tier": "enhanced",
          "tag" : "foo",
          "callback" : CALLBACK_URL
        })
        
#multiple tags
response = await dg_client.transcription.prerecorded(source,
        { "model": "general", 
          "language": "en", 
          "tier": "enhanced",
          "tag" : ["foo", "bar"],
          "callback" : CALLBACK_URL
        })
```

These tags appear on the callback in the `metadata.tags` field
